pybamm/models/submodels/interface/open_circuit_potential/wycisk_ocp.py

```diff
@@ -83,11 +97,34 @@ def get_coupled_variables(self, variables):
                 ocp_bulk_eq
             )
 
-            H = self.phase_param.hysteresis(sto_surf)
+            inputs = {f"{Domain} {phase_name}particle stoichiometry": sto_surf}
+            lith_ref = pybamm.FunctionParameter(
+                f"{self.phase_param.phase_prefactor}{Domain} electrode lithiation OCP [V]",
+                inputs,
+            )
+            delith_ref = pybamm.FunctionParameter(
+                f"{self.phase_param.phase_prefactor}{Domain} electrode delithiation OCP [V]",
+                inputs,
+            )
+            H = abs(delith_ref - lith_ref) / 2
             variables[f"{Domain} electrode {phase_name}OCP hysteresis [V]"] = H
 
+            # determine dQ/dU
+            def Q(sto, epsilon_s_av):
+                """Capacity change as a function of stoichiometry and active material volume fraction"""
+                c_max = self.phase_param.c_max
+                # epsilon_s_av = self.epsilon_s_av
+                epsilon_s_av = pybamm.yz_average(epsilon_s_av)
+                V_electrode = self.phase_param.main_param.A_cc * self.domain_param.L
+                Li_max = c_max * V_electrode * epsilon_s_av
+                Q_max = Li_max * self.phase_param.main_param.F / 3600
+                return Q_max * sto
```

Comment:
Sorry this review process never ends, now this is duplicate of https://github.com/pybamm-team/PyBaMM/blob/fadf094d17d0562319e4d5065a2eee1a9848b004/pybamm/models/submodels/active_material/base_active_material.py#L70 (and also it looks like you defined the `Q` function twice, is there a difference I missed or was it an accident?)

Reply:
Yes, I tried to use the other submodel, but it looks like it wasn't defined before this submodel, so that variable wasn't accessible. Do you know how to ask for variables from other submodels before they are created?

Reply:
I think I just got it to work, not sure what error I was running into earlier.